<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Player</class>
 <widget class="QMainWindow" name="Player">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Media Player</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>391</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" columnstretch="1,0,0,0,0">
     <item row="1" column="1">
      <widget class="QPushButton" name="Play">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="5">
      <widget class="QSlider" name="Progress">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="Pause">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="Stop">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QSlider" name="Volume">
         <property name="value">
          <number>50</number>
         </property>
         <property name="sliderPosition">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="volumeLabel">
         <property name="text">
          <string>Volume</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>once</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>order</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>repeat once</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QProgressBar" name="spec125">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>30</width>
      <height>115</height>
     </rect>
    </property>
    <property name="value">
     <number>40</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="spec250">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>30</width>
      <height>115</height>
     </rect>
    </property>
    <property name="value">
     <number>40</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="spec500">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>30</width>
      <height>115</height>
     </rect>
    </property>
    <property name="value">
     <number>40</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="spec1000">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>30</width>
      <height>115</height>
     </rect>
    </property>
    <property name="value">
     <number>40</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="spec2000">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>30</width>
      <height>115</height>
     </rect>
    </property>
    <property name="value">
     <number>40</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="spec4000">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>30</width>
      <height>115</height>
     </rect>
    </property>
    <property name="value">
     <number>40</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="spec8000">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>30</width>
      <height>115</height>
     </rect>
    </property>
    <property name="value">
     <number>40</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="spec16000">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>30</width>
      <height>115</height>
     </rect>
    </property>
    <property name="value">
     <number>40</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="spec20000">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>30</width>
      <height>115</height>
     </rect>
    </property>
    <property name="value">
     <number>40</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="add">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>85</width>
      <height>113</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="remove">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>remove</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>2</y>
      <width>311</width>
      <height>151</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QProgressBar" name="spec63">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>30</width>
      <height>115</height>
     </rect>
    </property>
    <property name="value">
     <number>40</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="Next">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>280</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="previous">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>281</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Previous</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentSong">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>406</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/open.png</normaloff>:/images/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open an audio file</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/play.png</normaloff>:/images/icons/play.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="toolTip">
    <string>Play music</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/stop.png</normaloff>:/images/icons/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop music</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/pause.png</normaloff>:/images/icons/pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
